docs/backend-api/resources/fleet/driver_journal/entry.md

```diff
@@ -205,7 +205,7 @@ Updates driver journal entry. Only two fields `type` and `comment` are available
 ```
 
 #### errors
-
+* 201 – Not found in the database - if there is no entry in the driver journal with this id.
```

Comment:
If entry missed backend returns 204 - "Entity not found". 201 - if tracker not found.